### PR TITLE
Adds documentation to protobuf wrapper classes (XRP component)

### PR DESCRIPTION
**Pull request by amiecorso**
## High Level Overview of Change
Adds documentation based on the information in xrpl.org docs to the fields and constructors in native Swift protobuf wrapper classes. 

### Context of Change
The Java SDK is well documented in this capacity - JS and Swift need to be made consistent. 

### Type of Change

- [x] Documentation Updates

## Test Plan
None

## Future Tasks

Other two SDKs done

### CODE REVIEW

**Comment by austin-king** [comment]
> Documentation looks good, but looks like triple slash directives are not necessary for all of these comments?
> https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html

I also have never used /// and just googled it though so I might just be misunderstanding their use case

**Comment by hbergren** [comment]
@austin-king , this is Swift code, not TypeScript.

File: XpringKit/XRPPayment.swift [hunk 1]
```diff
@@ -5,11 +5,24 @@ import Foundation
 public struct XRPPayment: Equatable {
   // TODO(keefertaylor): Modify this object to use X-Address format.
 
+  /// The amount of currency to deliver.
   public let amount: XRPCurrencyAmount
+
+  /// The unique address of the account receiving the payment.
   public let destination: Address
+
+  /// (Optional) Arbitrary tag that identifies the reason for the payment.
```
Comment (hbergren) [inline XpringKit/XRPPayment.swift:14]: Tags don't identify the reason for the payment, they indicate which user to send the payment to.

For example, Coinbase probably has one (or a handful) of actual XRP addresses, but each Coinbase user gets a tag number. So you send the XRP to Coinbase, and then Coinbase uses the tag number to credit the respective user inside their system.

File: XpringKit/XRPTransaction.swift [hunk 2]
```diff
@@ -3,22 +3,56 @@ import Foundation
 /// A transaction on the XRP Ledger.
 ///
 /// - SeeAlso: https://xrpl.org/transaction-formats.html
-// TODO(keefertaylor): Modify this object to use X-Address format.
+/// TODO(keefertaylor): Modify this object to use X-Address format.
```
Comment (hbergren) [inline XpringKit/XRPTransaction.swift:6]: Q: Are all comments in Swift `///`? Even TODOs?

Comment (amiecorso) [reply XpringKit/XRPTransaction.swift]: Good question - I learned that no they're not:  multiline comments with `/** **/` and single line comments with `///` will show up in the generated interface, whereas multis with `/* */` and singles with just `//` won't.  This has been reverted.

File: XpringKit/XRPPayment.swift (hunk 1 above)
Comment (amiecorso) [reply XpringKit/XRPPayment.swift]: I've updated this comment across all three SDKs... Check out the new version?  It's based on the documentation on xrpl.org docs here (under destinationTag): https://xrpl.org/payment.html.  Do you think that works, or should I entirely get rid of the bit about reasons?

**Comment by austin-king** [comment]
> @austin-king , this is Swift code, not TypeScript.

hahaha I'm an idiot

File: XpringKit/XRPPayment.swift (hunk 1 above)
Comment (hbergren) [reply XpringKit/XRPPayment.swift]: I think it's reasonable to use that language if that's what `xrpl.org` says.

File: XpringKit/XRPPathElement.swift [hunk 3]
```diff
@@ -1,7 +1,20 @@
 /// A path on the XRPLedger
 /// - SeeAlso: https://xrpl.org/paths.html#path-specifications
 public struct XRPPathElement: Equatable {
+
+  /// If present, this path step represents rippling through the specified address.
```
Comment (keefertaylor) [inline XpringKit/XRPPathElement.swift:5]: It's odd that You prefix these with 'this field is optional' and you use '(optional)' on the third? Any reason to not use consistent formatting?

Comment (amiecorso) [reply XpringKit/XRPPathElement.swift:5]: Nope - I think that is a relic of grabbing comments directly from xrpl.org, in which some of them use an "(Optional)" at the beginning and others don't.  I updated a bunch of this in `xpring4j` while explicitly transforming fields to `Optional` types so I'll actually clean this up here real quick.

**Pull request merged**